local-tests/setup/session-sigs/get-eoa-session-sigs.ts

```diff
@@ -29,9 +29,12 @@ export const getEoaSessionSigs = async (
   person: TinnyPerson,
   resourceAbilityRequests?: LitResourceAbilityRequest[]
 ) => {
-  if (devEnv.litNodeClient.config.litNetwork === LitNetwork.Manzano) {
+  const centralisation =
+    CENTRALISATION_BY_NETWORK[devEnv.litNodeClient.config.litNetwork];
+
+  if (centralisation === 'decentralised') {
```

Comment:
Not a blocker for us to land this since it's internal to our SDK, but I think this is more of a coincidental truth than an actual truth; really we probably want an entirely new map tracking e.g. `RLI_TOKEN_REQUIRED_BY_NETWORK`.  Also, when using `LOCALCHAIN` this could be different depending on how the nodes on the localchain are configured so I think we'll need to be able to set it from the environment cc: @joshLong145